db/migration/20201010162350_add_album_size.go

```diff
@@ -0,0 +1,31 @@
+package migration
+
+import (
+	"database/sql"
+
+	"github.com/pressly/goose"
+)
+
+func init() {
+	goose.AddMigration(Up20201010162350, Down20201010162350)
+}
+
+func Up20201010162350(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+alter table album
+	add size integer default 0 not null;
+	`)
+	return err
+	/*  IS A RESCAN NECESSARY TO UPDATE ALBUM TABLE WITH SIZES?
```

Comment:
Yes, the rescan will be necessary, as the sizes will be collected during the scan process